plugin/evm/gossiper_eth_gossiping_test.go

```diff
@@ -0,0 +1,375 @@
+// (c) 2019-2021, Ava Labs, Inc. All rights reserved.
```

Comment:
nit: change file name to tx_gossiping_test

Reply:
done